asctec_hl_comm/src/pressure_point_publisher.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+import rospy
+from geometry_msgs.msg import PointStamped
+from asctec_hl_comm.msg import mav_imu
+
+def callback(data):
+    #height = data.differential_height
+    height = data.height
+    
+    rospy.loginfo(rospy.get_caller_id() + "Pressure: %f", height)
+    pt = PointStamped()
```

Comment:
I prefer no abbreviations in variable names,  but since this is only temporary, this is fine with me.